src/config.rs

```diff
@@ -127,11 +129,15 @@ impl Config {
         }
 
         if matches.is_present("x") {
-            config.skip_validate_shark = true;
+            config.skip_validate_sharks = true;
         }
 
         config.domain = matches.value_of("domain").unwrap().to_string();
-        config.shark = matches.value_of("shark").unwrap().to_string();
+        config.sharks = matches
+            .values_of("shark")
+            .unwrap()
```

Comment:
This looks safe to call unwrap() on, but just wondering, could it panic?

Reply:
The unwrap() is pre-existing code.  But either way, clap should catch that because it requires that at least one `--shark` is specified.  See line 69 above.